### PR TITLE
Adding enos_config.py action plugin to create backup properly

### PR DESCRIPTION
**Pull request by amuraleedhar**
##### SUMMARY
This action plugin created for enos_config to create a folder and appropriate file for the config backup operation. The file name will bear the ip address and date and time stamp on it. Inside it will have the running config the time.
This is an issue reported by Ganesh of Ansible team during a review.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/plugins/action/enos_config.py

##### ANSIBLE VERSION
ansible 2.5.0
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible-2.5.0-py2.7.egg/ansible
executable location = /usr/local/bin/ansible
python version = 2.7.6 (default, Oct 26 2016, 20:30:19) [GCC 4.8.4]

##### ADDITIONAL INFORMATION
This is an effort to support Lenovo ENOS Switches in Ansible

### CODE REVIEW

**Comment by mkrizek** [comment]
cc @ganeshrn

**Review by ganeshrn — APPROVED** [review]

File: lib/ansible/plugins/action/enos_config.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
```
Comment (gundalow) [inline lib/ansible/plugins/action/enos_config.py]: This should be GPL
Can you also please update `enos.py`

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/action/enos_config.py (hunk 1 above)
Comment (amuraleedhar) [reply lib/ansible/plugins/action/enos_config.py]: Changing it to GPL for this file in this PR. The enos.py will be done in another PR

Comment (amuraleedhar) [reply lib/ansible/plugins/action/enos_config.py]: Will do it in another PR

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel`

**Pull request merged**